### ISSUE TITLE
Hyperparameters for baseline methods

### ISSUE THREAD

**Issue by futakw** [AI-secure/Transferability-Reduced-Smooth-Ensemble#2]
Did you set the exact same hyperparameters for baseline methods?

For example, batch size and learning rate scheduling seem to be set all the same for baselines in the code, although they are originally different in their original papers.

**Comment by Lucas110550** [comment]
Yes. Please check their original paper for baseline hyperparameters setting.
